### PR TITLE
Change display mode to add a comma when displaying surname first: "Lastname, Firstname"

### PR DESCRIPTION
**Pull request by JamesL813**
"Lastname Firstname" -> "Lastname, Firstname"

In response to this request:
#694

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/contacts/pro/models/Contact.kt [hunk 1]
```diff
@@ -108,7 +108,7 @@ data class Contact(
 
     fun getNameToDisplay(): String {
         val firstMiddle = "$firstName $middleName".trim()
-        val firstPart = if (startWithSurname) surname else firstMiddle
+        val firstPart = if (startWithSurname) if (surname.isNotEmpty() && firstMiddle.isNotEmpty()) "$surname," else surname else firstMiddle
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/contacts/pro/models/Contact.kt:111]: this got too long now, split the if/else into multiple lines with brackets please

Comment (JamesL813) [reply app/src/main/kotlin/com/simplemobiletools/contacts/pro/models/Contact.kt]: added brackets and split up lines for readablilty

File: app/src/main/kotlin/com/simplemobiletools/contacts/pro/models/Contact.kt [hunk 2]
```diff
@@ -108,7 +108,10 @@ data class Contact(
 
     fun getNameToDisplay(): String {
         val firstMiddle = "$firstName $middleName".trim()
-        val firstPart = if (startWithSurname) if (surname.isNotEmpty() && firstMiddle.isNotEmpty()) "$surname," else surname else firstMiddle
+        val firstPart = if (startWithSurname) {
+            if (surname.isNotEmpty() && firstMiddle.isNotEmpty()) {
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/contacts/pro/models/Contact.kt:112]: add new lines too, this is unreadable

Comment (JamesL813) [reply app/src/main/kotlin/com/simplemobiletools/contacts/pro/models/Contact.kt:112]: added more lines/formatting

**Comment by tibbi** [comment]
should be good, thanks

**Comment by TaivasJumala** [comment]
For Chinese, Japanese and Koreans, it's normal that they put surname first and they don't add comma.
Need to detect their languages.

**Pull request merged**